dockertest/output/dockerinfo.py

```diff
@@ -0,0 +1,114 @@
+"""
+Handler for 'docker info'. Runs command, returns output in
+machine-friendly form.
+"""
+
+import subprocess
+
+
+def _normalize(key):
+    """
+    Storage Driver -> storage_driver
+    """
+    return key.lower().replace(' ', '_')
+
+
+class DockerInfo(object):
+    """
+    Parser of 'docker info' output
+    """
```

Comment:
Because of the way the docs-parser works, the ``__init__()`` arguments should go into the class docstring.  I'm not going to block the merge on having this though...